build-system/tasks/validate-html-fixtures.js

```diff
@@ -72,13 +72,35 @@ async function getFileGroups(filesToCheck) {
   return fileGroups;
 }
 
+/**
+ * Checks for the existence of a local wasm / js validator binary and returns
+ * its location. Defaults to the wasm binary on the CDN.
+ * @return {Promise<string>}
+ */
+async function getValidatorJs() {
+  const localWasmValidatorPath =
+    'validator/bazel-bin/cpp/engine/wasm/validator_js_bin.js';
+  const localJsValidatorPath = 'validator/dist/validator_minified.js';
+  if (await pathExists(localWasmValidatorPath)) {
+    log('Using the', cyan('locally built wasm validator') + '...');
+    return localWasmValidatorPath;
+  }
+  if (await pathExists(localJsValidatorPath)) {
+    log('Using the', cyan('locally built js validator') + '...');
+    return localJsValidatorPath;
+  }
+  log('Using the', cyan('wasm validator from the CDN') + '...');
+  return 'https://cdn.ampproject.org/v0/validator_wasm.js'; // eslint-disable-line local/no-forbidden-terms
```

Comment:
not sure where would be most appropriate but could we leave a comment about when the other validators would not be available?

Reply:
Added in [`19baddd` (#37569)](https://github.com/ampproject/amphtml/pull/37569/commits/19baddd84cc8579f323aa0b83970575825e35cf3).